src/victory-util/events.js

```diff
@@ -120,7 +120,7 @@ export default {
         const childState = baseState[childName] || {};
 
         const filterState = (state) => {
-          const stateTargets = Object.keys(state[key]);
+          const stateTargets = state[key] && Object.keys(state[key]) || [];
```

Comment:
I think this reads better as:

```js
state[key] ? Object.keys(state[key]) : [];
```

so you don't need to remember precedence, etc. Also, because `state[key] &&` is a _test_ and not targeting creating a boolean value.